src/adapter/src/client.rs

```diff
@@ -254,6 +262,57 @@ Issue a SQL query to get started. Need help?
             )));
         }
 
+        // Users stub their toe on their default cluster not existing, so we provide a notice to
+        // help guide them on what do to.
+        let cluster_var = session
+            .vars()
+            .inspect(CLUSTER.name())
+            .expect("cluster should exist");
+        if catalog.resolve_cluster(Some(&cluster_active)).is_err() {
+            let cluster_notice = 'notice: {
+                // If the user provided a cluster via a connection configuration parameter, do not
+                // notify them if that cluster does not exist.
+                if cluster_var.inspect_session_value().is_some() {
+                    break 'notice None;
+                }
+
+                let role_default = catalog.get_role(catalog.active_role_id());
+                let role_cluster = match role_default.vars().get(CLUSTER.name()) {
+                    Some(OwnedVarInput::Flat(name)) => Some(name),
+                    None => None,
+                    // This is unexpected!
+                    Some(v @ OwnedVarInput::SqlSet(_)) => {
+                        tracing::warn!(?v, "SqlSet found for cluster Role Default");
+                        break 'notice None;
+                    }
+                };
+
+                let alter_role = "with ALTER ROLE <role> SET cluster TO <cluster>";
```

Comment:
```suggestion
                let alter_role = "with `ALTER ROLE <role> SET cluster TO <cluster>;`";
```

This might be slightly clearer to users.

Reply:
Updated!